doc/user/content/sql/isolation-level.md

```diff
@@ -0,0 +1,87 @@
+SQL-transaction executes to completion before the next one begins. There is no guarantee that this serial ordering is
+consistent with the real time ordering of the transactions, in other words transactions are not linearizable under the
+Serializable isolation level. For example if SQL-transaction T1 happens before SQL-transaction T2 in real time, then the
+result may be equivalent to a serial order where T2 was executed first.
+
+In practice non-linearizable orderings are unlikely to happen, but still possible, when querying directly from tables
+and sources. Non-linearizable orderings are likely to surface when querying from indexes and materialized views with
+large propagation delays. For example, if SQL-transaction T1 happens before SQL-transaction T2 in real time, T1 queries
+table t, and T2 queries materialized view mv where mv is an expensive materialized view including t, then T2 may not see
+all the rows that were seen by T1 if they are executed close enough together in real time.
+
+## Strict Serializable
+
+The Strict Serializable isolation level provides all the same guarantees as Serializable, with the addition that
+transactions are linearizable. That means that if SQL-transaction T1 happens before SQL-transaction T2 in real time,
+then the execution is equivalent to a serial execution where T1 comes before T2.
+
+For example, if SQL-transaction T1 happens before SQL-transaction T2 in real time, T1 queries table t, and
+T2 queries materialized view mv where mv is an expensive materialized view including t, then T2 is guaranteed to see all
+of the rows that were seen by T1.
+
+It’s important to note that the linearizable guarantee only applies to transactions (including single statement SQL
+queries which are implicitly single statement transactions), not to data written while ingesting from upstream sources.
+So if some piece of data has been fully ingested from an upstream source, then it is not guaranteed to appear in the
+next read transaction. See https://github.com/MaterializeInc/materialize/issues/11531
+and https://github.com/MaterializeInc/materialize/issues/13107 for more details. If some piece of data has been
+fully ingested from an upstream source AND is included in the results of some read transaction THEN all subsequent read
+transactions are guaranteed to see that piece of data.
+
+## Choosing the Right Isolation Level
+
+It may not be immediately clear which isolation level is right for you. Materialize recommends to start with the default
+Strict Serializable isolation level. If you are noticing performance issues on reads, and your application does not need
+linearizable transactions, then you should downgrade to the Serializable isolation level.
+
+Strict Serializable provides stronger consistency guarantees but may have slower reads than Serializable. This is
+because Strict Serializable may need to wait for writes to propagate through materialized views and indexes, while
+Serializable does not.
+
```

Comment:
Here a **Learn more** could fit well.

E.g.: 
https://preview.materialize.com/materialize/14706/overview/architecture/#learn-more
https://preview.materialize.com/materialize/14706/overview/#learn-more

Reply:
Awesome, that works perfectly, I just updated it.